.github/actions/maven-build-and-tag/action.yml

```diff
@@ -21,6 +25,10 @@ inputs:
   maven-password:
     description: Nexus password
     required: true
+  maven-version:
```

Comment:
Curious: is it a temporary parameter needed for Poc?

Reply:
It is required to use Maven 3.9 with Maven Build Cache extension, but I did not want make it default

Reply:
What happen if value is not provided? Latest version will be set? I think it can be is description

Reply:
It will try to work with the provided Maven version, but the Maven build cache extension will back off if the Maven version is earlier than 3.9.x